debian/control

```diff
@@ -0,0 +1,20 @@
+Source: recovery-environment
+Section: misc
+Priority: optional
+Maintainer: Paul Dagnelie <pcd@delphix.com>
+Build-Depends: debhelper (>= 10), dropbear-bin
+Standards-Version: 4.1.2
+Vcs-Git: https://github.com/delphix/recovery-environment.git
+#Vcs-Browser: https://github.com/delphix/recovery-environment
+
+Package: recovery-environment
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
```

Comment:
This package copies a bunch of files from other packages in the postinst script (such as from the kernel). Therefore we need to make sure that it is installed after those other packages are installed, so we'd want to put those dependencies here. For instance, we would want to add `delphix-kernel` as a dependency here, as well as other packages that we are copying files from.